astropy/config/tests/test_configs.py

```diff
@@ -514,28 +515,20 @@ def get_content(fn):
     assert not is_unedited_config_file(content)
 
 
-class TestAliasRead:
-    def setup_class(self):
-        configuration._override_config_file = get_pkg_data_filename("data/alias.cfg")
+def test_alias_read():
+    from astropy.utils.data import conf
 
-    def test_alias_read(self):
-        from astropy.utils.data import conf
-
-        with pytest.warns(
+    with (
+        set_temp_config(Path(__file__).with_name("data") / "alias"),
```

Comment:
Are we not able to keep using `get_pkg_data_filename`?

Reply:
There is also the dir version:

https://github.com/astropy/astropy/blob/81ac1ed2b81dfe511ac839d1c13f77d6323ba43f/astropy/utils/data.py#L63

Just make maintenance easier if we use a common util. Getting rid of `get_pkg_data_xxx` usage is not in scope here.

Reply:
Maintenance is easier if we use the standard library.